pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/impl/UniformLoadShedder.java

```diff
@@ -154,8 +154,7 @@ public Multimap<String, String> findBundlesForUnloading(final LoadData loadData,
                                 }
                                 String bundle = e.getLeft();
                                 double bundleMsgRate = e.getRight();
-                                if (bundleMsgRate <= (msgRateRequiredFromUnloadedBundles.getValue()
-                                        + 1000/* delta */)) {
```

Comment:
what was the original intention for this 1000 delta? If removing this, would that cause any negative impact?

Reply:
Can we use a multiplication delta, like 1%, to keep this delta?

Reply:
I understand that delta=1000 here does not have much meaning. The initial design was just to allow the data of one shedding to reach a certain threshold and to prevent the data flow of one shedding from being too small.
However, later we added the configuration minUnloadMessage and minUnloadMessageThroughput. These two configurations can completely replace the role of delta=1000, so the delta here is meaningless. 
However, due to the existence of delta=1000, broker load balancing cannot be achieved in small traffic scenarios.

Reply:
I see. Plz update the motivation in the description that this threshold(delta) can be controlled by minUnloadMessage and minUnloadMessageThroughput. Hence , keeping this delta is redundant or error prone for the smaller traffic too.

Reply:
OK, I have updated motivation